packages/web3-utils/test/fixtures/stringManipulation.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+
+import { HexString, Numbers } from "../../src/types";
+
+
+export const padLeftData: [[Numbers, number, string], HexString][] = [
+    [[0, 10, '0'], '0x0000000000'],
+    [['0x01', 64, 'f'], '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01'],
+    [['-0x01', 64, 'f'], '-0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01'],
+    [['5', 32, '0'], '00000000000000000000000000000005'],
+    [['-05', 32, 'f'],'fffffffffffffffffffffffffffff-05'],
+    [['abcd', 8, '0'],'0000abcd'],
+    [['-abcd', 8, '0'],'000-abcd'],
+    [[BigInt('9007199254740992'),32,'0'],'0x00000000000000000020000000000000'],
+    [[BigInt('-9007199254740992'),32,'0'],'-0x00000000000000000020000000000000'],
+    [[-13, 10, '0'],'-0x000000000d'],
+    [['9.5', 8, '0'],'000009.5']
+]
+
+export const padLeftInvalidData: [[any, number, string], HexString][] = [
+    [[9.5, 64, 'f'], 'Invalid value given "9.5". Error: not a valid integer.'],
+    [[null, 8, '0'],'Invalid value given "null". Error: not a valid integer.'],
+    [[undefined,8,'0'], 'Invalid value given "undefined". Error: not a valid integer.'],
+    [[{},3, 'f'], 'Invalid value given "[object Object]". Error: not a valid integer.'],
+]
+
+export const padRightData: [[Numbers, number, string], HexString][] = [
+    [[1, 5, '0'], '0x10000'],
+    [[-2000, 128, '0'], '-0x7d000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'],
+    [['0x00', 5, 'f'], '0x00fff'],
+    [['-0x01', 64, 'f'], '-0x01ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff'],
+    [['zxy', 11, '0'],'zxy00000000'],
+    [['-abcd', 32, '1'],'-abcd111111111111111111111111111'],
+    [[BigInt(10000),8,'0'],'0x27100000'],
```

Comment:
```suggestion
    [[BigInt(10000),8,'0'],'0x27100000'],
    [[10000n,8,'0'],'0x27100000'],
```

Add test case for alternative syntax for BigInt